docs/source/configuration/advanced_configuration.md

```diff
@@ -1,137 +1,48 @@
 # Advanced configuration
 The documentation on [configuration](./configuration_basics.md) describes how to satisfy most common requirements of standard Kedro project configuration:
 
-By default, Kedro is set up to use the [ConfigLoader](/kedro.config.ConfigLoader) class. Kedro also provides two additional configuration loaders with more advanced functionality: the [TemplatedConfigLoader](/kedro.config.TemplatedConfigLoader) and the [OmegaConfigLoader](/kedro.config.OmegaConfigLoader).
-Each of these classes are alternatives for the default `ConfigLoader` and have different features. The following sections describe each of these classes and their specific functionality in more detail.
+By default, Kedro is set up to use the [OmegaConfigLoader](/kedro.config.OmegaConfigLoader) class.
```

Comment:
Pet peeve but any chance we can start using cross references more often?

```suggestion
By default, Kedro is set up to use the {py:class}`~kedro.config.OmegaConfigLoader` class.
```

https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#reference-roles

Reply:
> Pet peeve but any chance we can start using cross references more often?

That'll all come when we do the linking work in #3015 (we can fix each and every link). But I'm dreading it, what a horrid syntax 🤮

Reply:
BTW, should we update to use the myst syntax for page links `[my other page](../../page.md)` and anchors to subheads too, or just to API docs? @astrojuanlu

Reply:
I didn't want to suggest a sweeping change to page xrefs too (because I'm mindful the syntax is a bit weird), maybe we can start with API objects and see if we get used to it and start using it more?

Reply:
Also this syntax does not depend on intersphinx, mind you. It's the same whether the object being linked is on this project or another one @stichbury

Reply:
@stichbury @astrojuanlu Do I need to make these changes now or will this be a separate PR?

Reply:
I'd be in favour of replacing references to `OmegaConfigLoader` with this, but I defer the final call to @stichbury

Reply:
Let's do it as a separate PR so @ankatiyar can get this one out and we can make a set of changes wholesale later.